src/backend/core/tests/conftest.py

```diff
@@ -0,0 +1,148 @@
+"""Defines fixtures for the tests."""
```

Comment:
hope we don't abuse this pattern. I like tests to be self understandable without having to jump from file to file to try to understand each fixture.
+ the necessity to define edge cases tends to create complicated code when you try to mutualize test code.